packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -86,6 +86,20 @@ EOF
 %doc %{gem_instdir}/README.md
 %doc %{gem_docdir}
 
+%pre
+if [ -d %{foreman_proxy_dir}/.ansible ]
+  mv %{foreman_proxy_dir}/.ansible /var/lib/foreman-proxy/ansible
+  ln -sv /var/lib/foreman-proxy/ansible %{foreman_proxy_dir}/.ansible
```

Comment:
I think this symlink (and others) should be in ```%install``` and ```%files```. See https://github.com/theforeman/foreman-packaging/blob/d09a1169ef51b8854f873f203a9b6aca54baddf8/packages/foreman/foreman/foreman.spec#L1000-L1003 as well. Because this is in ```%pre``` I think it should be installed after this hook is ran.